schemas/ajv.absolutePath.js

```diff
@@ -16,13 +18,28 @@ module.exports = (ajv) => ajv.addKeyword("absolutePath", {
 	errors: true,
 	type: "string",
 	compile(expected, schema) {
-		function callback(data) {
-			const passes = expected === /^(?:[A-Za-z]:\\|\/)/.test(data);
-			if(!passes) {
+		const callback = expected ? data => {
+			if (path.isAbsolute(data)) {
+				if (data.indexOf("!") >= 0) {
```

Comment:
Relative paths are also not allowed to contain `!`.

Best stay with the original code and just add a test for `!` before the absolute path test.